.github/workflows/need-attention-label.yml

```diff
@@ -0,0 +1,22 @@
+name: Add need attention label
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  apply-label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v3
+        #assume the login of our bot called 'SAMCLIBOT'
+        if: github.event.sender.login != 'SAMCLIBOT'
```

Comment:
Where is `SAMCLIBOT` defined?

Reply:
We need a repo scope token to run this github action and we need a user account to create this token. Here I assume the login for that account is 'SAMCLIBOT'.